11/README.md

```diff
@@ -0,0 +1,92 @@
+```
+shortname: 11/PYTHON-ORM-DRIVER
+name: ORM Implementation For Official BigchainDB Python Driver
+type: Standard
+status: raw
+editor: Manan Patel <manan@bigchaindb.com>
+```
+
+# Abstract
+
+This is a proposal to implement ORM based wrapper for official bigchaindb [python driver](https://github.com/bigchaindb/bigchaindb-driver). Official driver provides bare bones pyhton API on top of HTTP API to connect to BigchainDB network, but ORM implementation's aim is to provide a CRAB based operational model on top of official python API in order to simplify operations on bigchainDB network - thereby improving community adoption and engagement.  
+
+## Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [BCP 14](https://tools.ietf.org/html/bcp14) \[[RFC2119](https://tools.ietf.org/html/rfc2119)\] \[[RFC8174](https://tools.ietf.org/html/rfc8174)\] when, and only when, they appear in all capitals, as shown here.
+
+# Motivation
+
+The need for this implementation is driven by following rationale -
+
+* To provide simpler API to interact and perform operations on bigchainDB. Since, traditionally ORM based CRUD (Create, Read, Update, Delete) are the most used basic operations for the database (SQL and NoSQL), CRAB model for blockchains (bigchainDB in this case) reduces the learning curve for non-blockchain developers and improves community engagement. This is specifically true for python developers as Django (one of the most popular python framework) incorporates ORM model to perform CRUD operations.
+* Our [ORM implementation](https://github.com/bigchaindb/js-driver-orm) of [javascript driver](https://github.com/bigchaindb/js-bigchaindb-driver) has recently shown good traction and usage within community.
+* We would like to keep consistency with implementations related to both - javascript and python, offical drivers.  
+
+# Specification
+
+This ORM implmentation is inspired from the ORM implementation of [bigchaindb javascript driver](https://github.com/bigchaindb/js-driver-orm).This ORM SHOULD be implemented on top of python driver - [https://github.com/bigchaindb/bigchaindb-driver](https://github.com/bigchaindb/bigchaindb-driver).
+This implementation SHOULD include following **CRAB** operations -
+
+1. **C**reate - performs CREATE transaction with asset payload into bigchainDB.
+
+```
+/**
+* creates the asset in bigchaindb
+* @param {dict} inputs - dictionary containing keypairs and data for the asset
+*/
+def create(inputs) 
```

Comment:
I find it very strange that each function/method has exactly one input and it's always a dict. Why not a list of arguments and/or keyword arguments?